src/abstract-ops/spec-types.mjs

```diff
@@ -15,7 +17,19 @@ import {
   OrdinaryObjectCreate,
   ToBoolean,
 } from './all.mjs';
+import { isNonNegativeInteger } from './data-types-and-values.mjs';
 
+// #𝔽
+export function 𝔽(x) {
+  Assert(typeof x === 'number');
+  return new NumberValue(x);
+}
+
+// #ℤ
+export function ℤ(x) {
+  Assert(typeof x === 'bigint');
+  return new BigIntValue(x);
+}
```

Comment:
can these just use `F` and `Z` instead?

Reply:
For the most part, yes, but there are a few cases where they need to be renamed due to conflicts with local variables named `F` (in `abstract‑ops/function-operations.mjs` and `abstract‑ops/regexp-objects.mjs`)